docs/topics/wildfly-swarm/docs/reference/fractions/io.adoc

```diff
@@ -24,16 +24,49 @@ How many buffers per slice, if not set optimal value is calculated based on avai
 swarm.io.buffer-pools._KEY_.direct-buffers:: 
 Does the buffer pool use direct buffers, some platforms don't support direct buffers
 
+swarm.io.workers._KEY_.core-pool-size:: 
+Minimum number of threads to keep in the underlying java.util.concurrent.ThreadPoolExecutor even if they are idle. Threads over this limit will be terminated over time specified by task-keepalive attribute.
```

Comment:
full class path should be formatted with backticks

Reply:
Note to self: not easily fixable in our sources.